.github/ISSUE_TEMPLATE/other-fr.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Autre
+about: Poser une question, laisser un commentaire, demander de l'aide
+title: "[OTHER] Résumé de la requête"
+labels: ''
+assignees: ''
+
+---
+
+## Objectif de cette requête
```

Comment:
```suggestion
## Objectif de cette requête
<!-- Obligatoire -->
```

Reply:
Having it in the preceding section header is sufficient.